### PR TITLE
Targets

### PR DESCRIPTION
**Pull request by randyhoulahan**
bug fixes
inline with 6th

### CODE REVIEW

File: app/directives/aichi-targets/side-events.js [hunk 1]
```diff
@@ -69,9 +69,18 @@ define(['app','lodash', 'text!./side-events.html','data/aichi-targets/targets','
...
+                    schedule: {$exists:true}
+                  }
+                  var sort = {
+                    schedule : {
+                      start:1
+                    }
+                  }
+                  var fields = { Title:1,code:1}
+                    return mongoStorage.loadDocs('conferences',q, 0, 1000,null,fields).then(function(res){
+                      $scope.confrences = res.data
```
Comment (sbilodeau) [inline app/directives/aichi-targets/side-events.js]: misspelled: `confrences`

Comment (sbilodeau) [inline app/directives/aichi-targets/side-events.js]: I don't know whet mongo storeage do but I guessing it reteurns document. I thinks `res.data` is wrong. it probably return document array directly

File: app/directives/aichi-targets/side-events.js [hunk 2]
```diff
@@ -175,7 +184,7 @@ define(['app','lodash', 'text!./side-events.html','data/aichi-targets/targets','
                   }
                   q['sideEvent.targets']={'$in':['AICHI-TARGET-'+$scope.target]};
                   q['meta.status']={'$nin':[ 'deleted', 'archived']};
-
+                  q['start']= {"$ne":null};
```
Comment (sbilodeau) [inline app/directives/aichi-targets/side-events.js]: if start is "not set" this condition won't work. is it ok?

Comment (randyhoulahan) [reply app/directives/aichi-targets/side-events.js]: $ne selects the documents where the value of the field is not equal to the specified value. This includes documents that do not contain the field. 
see: https://docs.mongodb.com/v3.6/reference/operator/query/ne/

**Pull request merged**